internal/jimm/access_test.go

```diff
@@ -1110,3 +1110,99 @@ func TestListGroups(t *testing.T) {
 	c.Assert(groups[3].Name, qt.Equals, "test-group1")
 	c.Assert(groups[4].Name, qt.Equals, "test-group2")
 }
+
+func TestOpenFGACleanup(t *testing.T) {
+	c := qt.New(t)
+	ctx := context.Background()
+
+	ofgaClient, _, _, err := jimmtest.SetupTestOFGAClient(c.Name())
+	c.Assert(err, qt.IsNil)
+
+	type createTagFunction func(int) *ofga.Entity
+
+	var (
+		createStringTag = func(kind openfga.Kind) createTagFunction {
```

Comment:
Something worthwhile for creating OpenFGA tuples would be to create a similar construct to our database `yaml` parser where we define a yaml string of what will go into our database. We could have the same for OpenFGA tuples, define it in a yaml string above the test and call some test function to create them.

Reply:
i could do, but i wanted to create an arbitrary number of tuples so a function seemed to make more sense